util/logging/retrieve-cluster-logs.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import gzip
+import json
+import logging
+import os
+import sys
+import tarfile
+import tempfile
+import time
+from datetime import datetime
+
+import argparse
+import boto3
+
+from pcluster.cloudwatch_logs.utils import get_log_group_name
```

Comment:
Same here. As long as this is a util script let's drop any dependency on the cli code.

Reply:
Done.